File: Easy/count_ones.py
# https://edabit.com/challenge/GbyPdqNnp75Wci7Cn

def count_ones(num):
	return bin(num).count("1")


print(count_ones(12), 2)
print(count_ones(0), 0)
print(count_ones(100), 3)
print(count_ones(101), 4)
print(count_ones(999), 8)
print(count_ones(123456789), 16)
print(count_ones(1234567890), 12)

File: javascript/Easy/countOnes.js
// https://edabit.com/challenge/ve7mQnJsjtFep97fm

function countOnes(m) {
    let total = 0
	for (let i = 0; i < m.length; i++) {
        for (let j = 0; j < m[i].length; j++) {
            if (m[i][j]==1) {
                total++;
            }
        }
    } return total
}   